docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,197 @@
 ---
-    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+   description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
 ---
 # Quick Start Installation Guide
 
 In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
 
+Different options for installation include:
+
+* [Quick install (recommended!)](#quick-install)
+* [Install via Minikube](#minikube-users)
+* [Install in CI](#install-in-ci)
+* Or, you can [install manually](/user-guide/manual-install)
+
 ## Before You Begin
 
 The Ambassador Edge Stack is designed to run in Kubernetes for production. The most essential requirements are:
 
 * Kubernetes 1.11 or later
 * The `kubectl` command-line tool
+* [Edge Control](/reference/edgectl-download)
+
+## Quick Install
+
+The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally. Or, if you're a Minikube user, [check out these directions](#minikube-users).
 
-## Install the Ambassador Edge Stack
+When you use Edge Control to install the Ambassador Edge Stack on your publicly
+accessible cluster, it will:
 
```

Comment:
The first thing it will do is install AES, ja? I think that should be item one on the list.
Also these should be numbered 1, 2, 3, 4...